sat-plugin/src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -214,6 +214,12 @@
       <property name="severity" value="info"/>
       <property name="allowMissingPropertyJavadoc" value="true"/>
     </module>
+    
+    <module name="org.openhab.tools.analysis.checkstyle.AuthorContributionDescriptionCheck">
+       <property name="severity" value="warning"/>
+       <property name="requiredContributionDescriptions" value="initial contribution,initial contribution and api"/>
```

Comment:
Since you are checking with a "contains", it should be enough to list "initial contribution" here, since the other string is then automatically covered as well, right?

Reply:
Yes, you are absolutely right! Thanks!